app/api/entities/db_connection.rb

```diff
@@ -0,0 +1,15 @@
+module Entities
+  class DbConnection < Grape::Entity
+    expose :id
+    expose :user_id
+    expose :name
+    expose :db_type
+    expose :host
+    expose :port
+    expose :password_encrypted
+    expose :iv
+    expose :init_db
```

Comment:
does `init_db` denote the path to an initial database? If yes, would it make sense to rename this attribute?

Reply:
exactly, `inital_database` more expressive?